R/cacheName.R

```diff
@@ -36,7 +36,7 @@ cacheName <- function(prefix, type, args = NULL,  graph = NULL, mode = "put", pa
 
   fp <- fingerprint(name = paste0(fpprefix, type), graph = graph, details = (mode == "put"),
                     packages = packages, globalenv = globalenv)
-  if (identical(fp, "fingerprintError") && !isTRUE(getConfig("forcecache"))) {
+  if (fp == "fingerprintError" && !isTRUE(getConfig("forcecache"))) {
```

Comment:
This will fail if `fp` is `NULL`. one could perhaps use `as.character` to drop the attributes:

```suggestion
  if (identical(as.character(fp), "fingerprintError") && !isTRUE(getConfig("forcecache"))) {
```